charts/jaeger/templates/allinone-deploy.yaml

```diff
@@ -123,6 +133,11 @@ spec:
         {{- toYaml .Values.allInOne.podSecurityContext | nindent 8 }}
       serviceAccountName: {{ template "jaeger.fullname" . }}
       volumes:
+      {{ if .Values.userconfig }}
+         - name: jaeger-config
+           configMap:
+            name: jaeger-configmap
```

Comment:
please clean-up all the naming. Not jaeger-config, not jaeger-configmap, not `allInOne-config.yaml`, not `config.yaml` - everything should be using a single name `user-config`

Reply:
done should i convert it to PR